app/interactors/send_team_hours_update.rb

```diff
@@ -0,0 +1,15 @@
+class SendTeamHoursUpdate
+  include Interactor
+  include HasHarvest
+
+  def call
+    Team.active.each do |team|
+      from = Date.this_week.first
+      to = Date.this_week.last
```

Comment:
I think in the other place that iterates through users, we build the range inside the iteration because we need the user's time zone.